data/en/2023/chinese/committee.yml

```diff
@@ -1,30 +1,30 @@
 items:
   - name: "Eric Meng"
-    title: "WG Member(Primeton)"
```

Comment:
This is the English version of the Chinese event's committee. I think the `title` changes should be reverted here.

Reply:
Just fixed this in [06d1bf0](https://github.com/jakartaee/jakartaone.org/pull/499/commits/06d1bf0519ad8685e904171f7357a48e89336c53)